src/store/ChartStore.js

```diff
@@ -419,6 +419,14 @@ class ChartStore {
             params = { periodicity: calculateTimeUnitInterval(granularity) };
         }
 
+        if (params === undefined && symbolObj) {
+            for (const field in this.stxx.chart.series) {
+                if (this.stxx.chart.series[field].parameters.bucket !== 'study') {
```

Comment:
@brucebinary It looks fine and works fine, thanks! except I don't think chartiq add series for indicators

Reply:
@MahboobehMohammadi this line is from their original source code some versions back. It could be outdated though.